ote/src/cljs/ote/views/pre_notices/listing.cljs

```diff
@@ -70,38 +70,45 @@
     (let [pre-notices (filter #(= (::t-operator/id transport-operator)
                                   (::t-operator/id %))
                               pre-notices)]
-      [:div
-       [:div {:style {:margin-bottom "20px"}}
-        [list-header/header
-         app
-         (tr [:pre-notice-list-page :header-pre-notice-list])
-         [ui/raised-button {:id "add-new-pre-notice"
-                            :label (tr [:buttons :add-new-pre-notice])
-                            :on-click #(do
-                                         (.preventDefault %)
-                                         (e! (pre-notice/->CreateNewPreNotice)))
+      (if (:transport-operator app)
+        [:div
+         [:div {:style {:margin-bottom "20px"}}
+          [list-header/header
+           app
+           (tr [:pre-notice-list-page :header-pre-notice-list])
+           [ui/raised-button {:id "add-new-pre-notice"
+                              :label (tr [:buttons :add-new-pre-notice])
+                              :on-click #(do
+                                           (.preventDefault %)
+                                           (e! (pre-notice/->CreateNewPreNotice)))
+                              :primary true
+                              :icon (ic/content-add)}]
+           [t-operator-sel/transport-operator-selection e! app]]]
+         [:div {:style {:margin-bottom "40px"}}
```

Comment:
Repeated hard-coded values

Reply:
Yes there are few. But in this stage there must be because pre-notices list and editor is not converted fully to "wide" ui style yet. We are getting there slowly :)